docs/introduction.org

```diff
@@ -0,0 +1,106 @@
+* Introduction
+
+Nowadays, multiple languages have emerged to compete with the well-established C language.
+In fact, C is widely used in many places, learned by a lot of engineers, and used for many purposes.
+C is a powerful tool, /but with great power comes great responsibility/, in fact, memory leaks can easily occur by mistake.
```

Comment:
/but ?

Reply:
This will italicise the text, check the markdown view